tools/gulp/tasks/unit-test.ts

```diff
@@ -48,20 +48,20 @@ gulp.task(':test:deps:inline', sequenceTask(':test:deps', ':inline-resources'));
  *
  * This task should be used when running unit tests locally.
  */
-gulp.task('test', [':test:watch'], (done: () => void) => {
+gulp.task('test', [':test:watch'], () => {
   new karma.Server({
     configFile: path.join(PROJECT_ROOT, 'test/karma.conf.js')
-  }, done).start();
+  }).start();
```

Comment:
I can switch it to:

```js
() => {
  done();
  process.exit();
}
```

I didn't include it initially, because it seemed redundant since `process.exit` kills the process anyway (which includes Gulp).